web/client/localConfig.json

```diff
@@ -282,23 +282,11 @@
           },
           "collapsible": false
         },
-        "excludeOptions": [
-            "search",
-            "links",
-            "allowedSRS",
-            "title",
-            "type",
-            "visibility",
-            "params",
-            "legend",
-            "statistics",
-            "columnDefs",
-            "advancedFilter",
-            "boundingBox",
-            "capabilities",
-            "describeCoverage",
-            "describeLayer",
-            "loading"
+        "includeOptions": [
```

Comment:
This options should be the defaults, so they should not be present in the localConfig.json, but overridable in this way if needed.